tests/fixtures/tag-on-parent/dot-git/objects/fb/26504da0ed5cd9ed366f7428c06a8433fd76e6

```diff
@@ -0,0 +1,2 @@
+x��A��0Eg�Sx��<�qS	!�p��q���A�ק�a��K��I������e�*P�>a�Ys��Q��>ƚU�Gr���l�̵S�%r(��F
+B5K�<uZ�T�.�6���==�҆Q8�����f4��>�4��M'���ā�C��u���o�{����^��p5����%&Y�
```

Comment:
i don't think you intended to include this file

Reply:
actually this file is required!